wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/CommandPartElementToWomCommandPart.scala

```diff
@@ -101,6 +101,15 @@ case class WdlomWomPlaceholderCommandPart(attributes: PlaceholderAttributeSet, e
           ).validNel
         case None => "Array value was given but no 'sep' attribute was provided".invalidNel
       }
+    case WomArray(WomArrayType(WomOptionalType(_)), arrayValue) =>
```

Comment:
I don't fully understand the addition of this, but does it mean that we are adding support for Array[Array[Optional]] type? 🤔

Reply:
Yes, this comes originally from this commit: https://github.com/henriqueribeiro/cromwell/pull/55/commits/d5b55dbbb11d051b4a772a91de383a68a5015c48

Part of "support for Array[File?] as job input/output"